relay-metrics/src/aggregation.rs

```diff
@@ -716,8 +716,26 @@ impl Bucket {
 
 /// Any error that may occur during aggregation.
 #[derive(Debug, Fail)]
-#[fail(display = "failed to aggregate metrics")]
-pub struct AggregateMetricsError;
+#[fail(display = "failed to aggregate metrics: {}", kind)]
```

Comment:
_nit_: not sure how this composes outside, but you might be able to go without the prefix?

Reply:
I can do without prefix but then I need to put the prefix into the log message, otherwise the sentry errors just print "invalid character" without context